apps/fishing-map/public/locales/source/translations.json

```diff
@@ -228,7 +228,7 @@
     "onlyVisibleForJAC": "Only visible for JAC users",
     "onScreen": "on screen",
     "or": "or",
-    "others": "Others",
+    "others": "Points",
```

Comment:
talked with @satellitestudiodesign and we think is better to maintain others as a generic category